aiosmtpd/tests/test_smtp.py

```diff
@@ -58,37 +69,70 @@ def factory(self):
 
 
 class PeekerHandler:
-    session: SMTPSess = None
+    sess: SMTPSession = None
 
     async def handle_MAIL(
             self, server, session, envelope, address, mail_options
     ):
-        self.session = session
+        self.sess = session
         return "250 OK"
 
+    async def auth_DENYMISSING(self, server, args):
+        return MISSING
+
+    async def auth_DENYFALSE(self, server, args):
+        return False
+
     async def auth_NULL(
             self, server, args
     ):
         return "NULL_login"
 
+    async def auth_NONE(self, server: Server, args):
+        await server.push("235 2.7.0  Authentication Succeeded")
+        return None
+
     async def auth_DONT(
             self, server, args
     ):
         return MISSING
 
 
 class PeekerAuth:
-    def __init__(self):
-        self.login = None
-        self.password = None
```

Comment:
No need for __init__ if all it does is initializing some instance attributes.

Just convert those instance attributes into class attributes; first time the instance attribute gets assigned to, an instance attribute will be created from the class attribute.